src/plugins/orangehrmPerformancePlugin/Controller/EmployeeTrackerLogsController.php

```diff
@@ -49,4 +53,16 @@ public function preRender(Request $request): void
 
         $this->setComponent($component);
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function isCapable(Request $request): bool
+    {
+        $id = $request->attributes->getInt('id');
+        if (!$this->getUserRoleManager()->isEntityAccessible(PerformanceTracker::class, $id)) {
+            throw new RequestForwardableException(NoRecordsFoundController::class . '::handle');
+        }
+        return true;
```

Comment:
```suggestion
        return $this->getUserRoleManager()->isEntityAccessible(PerformanceTracker::class, $id);
```
can you check this?
cz `throw new RequestForwardableException(NoRecordsFoundController::class . '::handle');` already handled in `preRender` method. two different scenarios. one is no record found, other one is permission check. permission check handling in `isCapable`

Reply:
added no records found check to `isCapable`. hope it's okay now